src/components/TextField.js

```diff
@@ -0,0 +1,25 @@
+import * as s from "./TextField.sc";
+
+export default function TextField({
```

Comment:
is it possible that we did not have a component for a `TextField`? Or `TextInput`?